types/types.go

```diff
@@ -44,6 +44,8 @@ const (
 	ValidatingWebhookName = "longhorn-webhook-validator"
 	MutatingWebhookName   = "longhorn-webhook-mutator"
 
+	DefaultRecoveryBackendServerPort = 9600
```

Comment:
Remember to update our doc about port usage.

Reply:
Done. Thank you.